api/client/src/components/share/Share.js

```diff
@@ -161,7 +161,7 @@ const Share = ({ user, addNewPost }) => {
               )}
               <SearchLocationModal
                 setLocationModalActive={setLocationModalActive}
-                setCheckInLocation={setCheckInLocation}
+                updateLocation={setCheckInLocation}
```

Comment:
Should the `setCheckInLocation` name be changed as well?

Reply:
SearchLocationModal is a reusable component , the name of one of the properties was updated to updateLocation to make it more generic. Which is why setCheckInLocation isn't renamed cus its context is within the Share component.